guides/release/components/helper-functions.md

```diff
@@ -302,70 +299,114 @@ Class helpers are useful when the helper logic is fairly complicated, requires
 helper makes it easy to dynamically look up a property on an object or an element in an array. The second argument to `{{get}}` can be a string or a number, depending on the object being accessed.
 
-
 To access a property on an object with a string key:
 
-```handlebars
-{{get this.someObject "object_key"}}
+```gjs
+import { get } from '@ember/helper';
+
+const someObject = { object_key: 'Value' };
+
+<template>
+  {{get someObject "object_key"}}
+</template>
 ```
 
 To access the first element in an array:
 
-```handlebars
-{{get this.someArray 0}}
+```gjs
+import { get } from '@ember/helper';
+
+const someArray = [ 'one', 'two' ];
+
+<template>
+  {{get someArray 0}}
+</template>
 ```
 
 To access a property on an object with a dynamic key:
 
-```handlebars
-{{get this.address this.part}}
-```
+```gjs
+import { get } from '@ember/helper';
 
-If the `part` getter returns "zip", this will display the result of `this.address.zip`.
-If it returns "city", you get `this.address.city`.
+const address = { city: 'Chicago', state: 'IL', zip: '60610' };
+let part = 'zip';
 
+<template>
+  {{get address part}}
+</template>
+```
+
+If the `part` is "zip", this will display the result of `address.zip`.
+If it's "city", you get `address.city`.
 
 ### The `concat` helper
 
 We mentioned above that helpers can be nested. This can be
 combined with different dynamic helpers. For example, the
-[`{{concat}}`](https://api.emberjs.com/ember/4.5.0/classes/Ember.Templates.helpers/methods/concat?anchor=concat)
+[`{{concat}}`](https://api.emberjs.com/ember/release/classes/@ember%2Fhelper/methods/concat?anchor=concat)
 helper makes it easy to dynamically send a number of parameters to a component
 or helper as a single parameter in the format of a concatenated string.
 
-```handlebars
-{{get this.foo (concat "item" this.index)}}
+```gjs
+import { concat, get } from '@ember/helper';
+
+const foo = { item1: 'One', item2: 'Two' };
+let index = 1;
+
+<template>
+  {{get foo (concat "item" index)}}
+</template>
 ```
 
-This will display the result of `this.foo.item1` when index is 1, and
-`this.foo.item2` when index is 2, etc.
+This will display the result of `foo.item1` when index is 1, and
+`foo.item2` when index is 2, etc.
 
 ### The `let` helper
 
 Now let's say your template is starting to get a bit cluttered and you want
 to clean up the logic in your templates. This can be achieved with the `let`
 block helper.
-The [`{{let}}`](https://api.emberjs.com/ember/4.5.0/classes/Ember.Templates.helpers/methods/let?anchor=let)
+The [`{{let}}`](https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers/methods/let?anchor=let)
 helper lets you create new bindings (or temporary variables) in your template.
 
 Say your template now looks like this:
 
-```handlebars
-Welcome back {{concat (capitalize this.person.givenName) ' ' (capitalize this.person.familyName)}}
+```gjs
+import { concat } from '@ember/helper';
```

Comment:
why import a helper that this section isn't talking about?

Let's just make an object or array

```gjs

<template>
  {{#let (Object givenName="George" familyName="Washington") as |person|}}
     {{person.givenName}} {{person.familyName}}
  {{/let}}
</template>

```